src/executables/util_service.c

```diff
@@ -1665,6 +1674,7 @@ process_server (int command_type, int argc, char **argv, bool show_usage, bool c
 		{
 		  break;
 		}
+
```

Comment:
Did you include line feed for readability?

Reply:
I'm gonna remove this line.